runner/src/server/plugins/engine/models/SummaryViewModel.ts

```diff
@@ -218,6 +218,9 @@ export class SummaryViewModel {
       sectionPages.forEach((page) => {
         for (const component of page.components.formItems) {
           const item = Item(request, component, sectionState, page, model);
+          const shouldOmit = item.options?.omit ?? false;
+
+          if (shouldOmit === true) continue;
```

Comment:
This can be rewritten as:
```suggestion
          if (shouldOmit) continue;
```